renovate.json

```diff
@@ -1,37 +1,37 @@
 {
-  "extends": ["config:base", ":disableDependencyDashboard", ":preserveSemverRanges"],
-  "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": ["after 9am every weekday", "before 5pm every weekday"],
-  "enabledManagers": ["npm", "nvm", "dockerfile", "docker-compose", "helmv3", "helm-values", "circleci"],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:node"],
+  "reviewers": ["neilmendum", "timoj58", "natclamp-moj"],
   "packageRules": [
     {
       "matchManagers": ["npm"],
-      "matchPackageNames": ["@types/node"],
-      "matchUpdateTypes": ["major"],
-      "enabled": false
-    },
-    {
-      "matchDatasources": ["docker"],
-      "allowedVersions": "16.15-bullseye-slim"
+      "rangeStrategy": "bump"
     },
     {
-      "matchDatasources": ["docker-compose"],
-      "matchPackageNames": ["bitnami/redis"],
-      "allowedVersions": "5.0"
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major NPM dependencies",
+      "groupSlug": "all-npm-minor-patch",
+      "stabilityDays": 3
```

Comment:
Have introduced `stabilityDays`

> Number of days required before a new release is considered stable.

and `groupName` - to group them together in a single PR once they are at least 3 days old - this should ensure we aren't continually swamped with PRs. I spent some time testing this config on https://github.com/ministryofjustice/renovate-typescript-poc/pulls but can see how we find it?